src/Swashbuckle.AspNetCore.SwaggerGen/SwaggerGenerator/IDocumentFilter.cs

```diff
@@ -14,17 +14,21 @@ public class DocumentFilterContext
         public DocumentFilterContext(
             IEnumerable<ApiDescription> apiDescriptions,
             ISchemaGenerator schemaGenerator,
-            SchemaRepository schemaRepository)
+            SchemaRepository schemaRepository,
+            string documentName = null)
```

Comment:
Using a default here to prevent breaking existing code, but I doubt consumers are ever constructing these types themselves. If you'd prefer an overload, that's certainly an option, too.